netty/src/main/java/io/grpc/netty/NettyClientStream.java

```diff
@@ -214,6 +214,7 @@ public void writeFrame(
     @Override
     public void cancel(Status status) {
       try (TaskCloseable ignore = PerfMark.traceTask("NettyClientStream$Sink.cancel")) {
+        framer().dispose();
```

Comment:
NettyClientStream shouldn't need to manage this resource. AbstractClientStream/AbstractStream should be what handle it, potentially with calls from the likes of NettyClientStream/OkHttpClientstream. I see the dispose() on AbstractClientStream, but it 1) doesn't dispose the framer and 2) is never called.

Reply:
Moved `framer().dispose()` to `AbstractClientStream`.